db/schema.rb

```diff
@@ -13,12 +13,11 @@
 
 ActiveRecord::Schema.define(version: 20110111224543) do
 
-  create_table "users", force: true do |t|
+  create_table "users", force: :cascade do |t|
```

Comment:
It a change in how rails generates the schema in Rails 4+ that got picked up because this is the first time we've changed the test database schema in a long time!